### DISCUSSION TITLE
Error to CONFIGURE my BACKGROUND-IMAGES!

### DISCUSSION THREAD

**Discussion by phantodev** [tailwindlabs/tailwindcss#4789] [Ideas]
module.exports = {
    theme: {
      extend: {
        backgroundImage: theme => ({
         'hero-pattern': "url('/img/hero-pattern.svg')",
        })
      }
    }
  }

I was trying to put my background-images on CONFIG but the VSCODES show this error to me!

'theme' is declared but its value is never read.ts(6133)
Replace `theme` with `(theme)`eslintprettier/prettier
(parameter) theme: any

Can you help me guys?

**Comment by olets** [comment]
> Replace `theme` with `(theme)` eslintprettier/prettier

This is saying your eslint configuration needs `backgroundImage: (theme) =>` instead of `backgroundImage: theme =>`. Did you try that?

> 'theme' is declared but its value is never read.ts(6133)

This is saying that you're passing in the variable `theme` but you don't use it. So instead of

```javascript
backgroundImage: theme => ({
  …
})
```

you can do

```javascript
backgroundImage: {
  …
}
```
